lib/utils/load-config-esm.mjs

```diff
@@ -0,0 +1,14 @@
+// ES module to load the config file and print it to stdout
```

Comment:
What's the purpose of that module?

Reply:
This script is started as new Node.js process with CLI flags `--loader ts-node/esm --es-module-specifier-resolution=node` and Node recognizes it as ES module. Then it loads the Serverless configuration, which in this case is also an ES module.